src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -44,21 +44,42 @@ class ProjectFileParser {
         FilenameUtils.removeExtension(file) + newExtension
     }
 
+    File getOutputPath() {
+        if (msbuild.destinationDir) {
+            project.file(msbuild.destinationDir)
+        }
+        if (properties.TargetDir) {
+            new File (properties.TargetDir)
+        }
+    }
+
     File getDotnetAssemblyFile() {
-        project.file(properties.TargetPath)

```

Comment:
it happens when only solutionFile is set, the first ProjectFileParser would be solution and has no TargetPath